README.md

```diff
@@ -260,5 +260,5 @@ To change the User-Agent used by Typhoeus:
 HTML::Proofer.new("out/", {
   :typhoeus => {
     :headers => { "User-Agent" => "Mozilla/5.0 (compatible; My New User-Agent" }
```

Comment:
@jonbartlett, the value of `User-Agent` lacks a closing parenthesis. Could you please add it to your PR?